examples/gantry/gantry.cc

```diff
@@ -0,0 +1,54 @@
+/*
+ * Note that there is currently an annoying bug with the gantry's camera.
+ * You have to open a TV tuner program *before* this example, so that the
+ * TV tuner card is set to PAL B mode (cf. NTSC) otherwise the display
+ * will be mangled. There is definitely a way to do this in one of the
+ * Windows video APIs (though seemingly not in OpenCV) and I'll add this
+ * in when I can be bothered. -- AD
+ */
```

Comment:
OpenCV video does somehow seem to be singularly terrible in many ways...Maybe add this as a bug